src/main/java/it/pagopa/transactions/domain/TransactionId.java

```diff
@@ -0,0 +1,6 @@
+package it.pagopa.transactions.domain;
+
+import it.pagopa.transactions.annotations.ValueObject;
+
+@ValueObject
+public record TransactionId(String value) {}
```

Comment:
What do you think about using `UUID` for value?

Reply:
Why not :-)